client/src/view/global-fabric-panel.tsx

```diff
@@ -72,50 +64,56 @@ export function GlobalFabricPanel({constructFabric, fabric, cancelSelection}: {
         operation(fabric)
     }
 
+    function saveFiles(f: TensegrityFabric): void {
+        const dateString = new Date().toISOString()
+            .replace(/[.].*/, "").replace(/[:T_]/g, "-")
+        const output = f.output
+        FileSaver.saveAs(extractJointBlob(output), `${dateString}-joints.csv`)
+        FileSaver.saveAs(extractIntervalBlob(output), `${dateString}-intervals.csv`)
```

Comment:
Is it not possible to combine joints and intervals into one file?

Reply:
Yes but it's not worth the effort right now.